tests/php/core-api/test_class.jetpack-core-api-module-endpoints.php

```diff
@@ -0,0 +1,198 @@
+<?php
+require_once dirname( __FILE__ ) . '/../lib/class-wp-test-rest-controller-testcase.php';
+require_once dirname( __FILE__ ) . '/../lib/class-wp-test-spy-rest-server.php';
+
+use function Patchwork\redefine;
+use function Patchwork\restore;
+
+class WP_Test_Jetpack_Core_Api_Module_Activate_Endpoint extends WP_Test_REST_Controller_Testcase {
+
+	public static $xmlrpc_response = true;
+
+	public static $current_user_can = true;
+
+	public $redefines = array();
+
+	public function setUp() {
+		require_once dirname( __FILE__ ) . '/../../../_inc/lib/class.core-rest-api-endpoints.php';
```

Comment:
The `JETPACK__PLUGIN_DIR` could be used here.
Last time pointing this but there are other places where it could and should be used for better readability.